build/templates/session.py/default_method.py.mako

```diff
@@ -7,6 +7,9 @@
     suffix = method_template['method_python_name_suffix']
 
     enum_input_parameters = helper.filter_parameters(f, helper.ParameterUsageOptions.INPUT_ENUM_PARAMETERS)
+    # TODO(marcoskirsch): Retrofit to call filter_parameters(function, parameter_usage_options)
```

Comment:
This is simple and easy to read, but should we be using helper functions for consistency?

Reply:
Yeah, Marcos is right that we should probably use filter_parameters. That's why I copied his TODO from the location where I copied the logic. I may go ahead and do that retrofit; didn't really have time for it last night. But I'm also not the biggest fan of the giant UsageOptions dictionaries, or at least how they're filled in. I might change _that_, too. But that larger retrofit _really_ doesn't belong in this PR.

Reply:
Yeah, I'm not a big fan of some of this codegen code, either, but I agree that I don't want to add more big changes to this PR.